css/preview.css

```diff
@@ -0,0 +1,20 @@
+.node-preview-container {
+  padding: 0;
+  background-color: lightgrey;
```

Comment:
Let's use one of our CSS variables for this - `--color-grey-light` or something like that.